lib/browser/api/web-contents.js

```diff
@@ -326,6 +326,19 @@ WebContents.prototype._init = function () {
     }
   })
 
+  this.on('-ipc-invoke', function (event, channel, args) {
+    event._reply = (result) => event.sendReply({ result })
+    event._throw = (error) => {
+      console.error(`Error occurred in handler for '${channel}':`, error)
+      event.sendReply({ error: error.toString() })
+    }
+    if (ipcMain._invokeHandlers.has(channel)) {
```

Comment:
This property is private, how does this work?

Reply:
it's `.js`, not `.ts`.